infrastructure/zk/src/status.ts

```diff
@@ -0,0 +1,227 @@
+import { Command } from 'commander';
+
+import { Pool } from 'pg';
+import { ethers } from 'ethers';
+import { assert } from 'console';
+
+// Postgress connection pool - must be intialized later - as the ENV variables are set later.
+let pool: Pool | null = null;
+
+const GETTER_ABI = [
+    'function getTotalBlocksCommitted() view returns (uint256)',
+    'function getTotalBlocksVerified() view returns (uint256)',
```

Comment:
nit: we are deprecating "Blocks" naming so better use `getTotalBatchesCommitted`, `getTotalBatchesVerified`